charts/openfga/values.yaml

```diff
@@ -41,7 +41,7 @@ securityContext:
 
 initContainer:
   repository: groundnuty/k8s-wait-for
-  tag: "v1.6"
+  tag: "v2.0"
```

Comment:
Is there any way we can define this as constant so that line 154 will also use the same value